src/gui_common/ChemicalEquation.cs

```diff
@@ -77,6 +85,22 @@ public bool ShowSpinner
         }
     }
 
+    public bool ProcessEnabled
+    {
+        get => toggleProcess.ButtonPressed;
+        set
+        {
+            if (value == lastToggle)
+                return;
+
+            EmitSignal(SignalName.ToggleProcessPressed, this);
```

Comment:
Shouldn't this trigger after the value is changed? Because otherwise code reading the event cannot inspect the latest state of ProcessEnabled?